.github/workflows/build.yml

```diff
@@ -0,0 +1,39 @@
+name: Run
+on:
+  push:
+  pull_request:
+    types: [opened]
+jobs:
+  Rust-Tests:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        toolchain: [stable, beta, nightly]
+    steps:
+      - name: Build Dogear
+        run: |
+          git init .
+          git remote add origin https://github.com/mozilla/dogear.git
+          git pull origin ${GITHUB_REF##*/}
+          cargo clean
+          cargo build
+      - name: Run Rust Tests
+        run: cargo test
+  CodeCov:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        toolchain: [stable]
+    steps:
+      - name: Build Dogear
+        run: |
+          git init .
+          git remote add origin https://github.com/mozilla/dogear.git
+          git pull origin ${GITHUB_REF##*/}
+          cargo clean
+          cargo build
+      - name: Get Code Coverage
+        run: |
+          cargo install cargo-tarpaulin -f
+          cargo tarpaulin --out Xml
+          bash <(curl -s https://codecov.io/bash)
```

Comment:
even though we haven't fixed a-s yet, I think we should avoid this `bash < curl` pattern that caused us so much grief - I think just creating a ticket to add coverage and removing this job seems fine for now.

(hrm - I see if was already doing that, but I still think it makes sense to not bother for now)

We are also not going the docs thang - should be get a ticket on file for that? If not (which is a reasonable answer if we don't think there's enough value), we should probably arrange to take the existing docs down under the theory that stale docs are worse than no docs?

Either way, this is all your call :)

Reply:
I created issues #67 and #68, but I was also curious how feasible consolidating dogear in application services is. Is this possible and worth creating a ticket for? I'm assuming this would be something we'd consider as or after we update our versioning/release cutting/repo structure approach.